lib/matplotlib/collections.py

```diff
@@ -293,6 +294,14 @@ def draw(self, renderer):
 
         if self._hatch:
             gc.set_hatch(self._hatch)
+            if self._hatch_color is not None:
+                try:
+                    gc.set_hatch_color(self._hatch_color)
+                except AttributeError:
+                    # if we end up with a GC that does not have this method
+                    import warnings
+                    warnings.warn("Your backend does not have support for "
```

Comment:
"Your backend doesn't support setting the hatch color" is clearer for me (might be because I am a non native english speaker).